searcher/searcher.go

```diff
@@ -367,6 +365,13 @@ func newSearcher(
 		SpecialFiles:    wd.SpecialFiles(),
 	}
 
+	var vcsDir string
+	if (wd.IsLocal()) {
```

Comment:
like in in vcs.Driver?
Then each driver (apart from local) will need to implement basically the same code to get the working dir.